src/lib/WireGuard.js

```diff
@@ -199,7 +205,7 @@ AllowedIPs = ${client.address}/32`;
     return `
 [Interface]
 PrivateKey = ${client.privateKey}
-Address = ${client.address}/24
```

Comment:
Not 100% sure if this was a wanted behavior. Previously the exported client config had the 24 subnet, even tough the wg.conf sets it to /32 (https://github.com/wg-easy/wg-easy/pull/797/files#diff-65b7aac2a6e96abf37ec0c4c0c24a4e94ac70fc6a0702d74f45f79b900716f79L114)

Reply:
Address is not the same as AllowedIPs

Reply:
This is the cidr for the client IP. It may therefore need to be replaced with your new variable.

Reply:
I get that Address != AllowedIPs, but if the peer on the "server" has an allowed IP of /32 for the peer (as in line 114), it does not expect the "client" interface to have a subnet of /24. Or am i missing something?

Reply:
The keyword allowed-ips is a list of addresses that will get routed to the peer. Make sure to specify at least one address range that contains the WireGuard connection's internal IP address(es). from: https://wiki.archlinux.org/title/WireGuard#Peer_setup

Reply:
Okay i did some more research into this. 
Even tough it shouldn't matter to much for the use case wg-easy provides, i will change this line back to 24. If some users have site-site networks they need the /24, for everyone else it should not matter.